qa/integration-tests/src/test/java/io/camunda/zeebe/it/startup/RootlessImageTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.startup;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.response.ProcessInstanceResult;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.test.util.testcontainers.ContainerLogsDumper;
+import io.camunda.zeebe.test.util.testcontainers.ZeebeTestContainerDefaults;
+import io.zeebe.containers.ZeebeBrokerContainer;
+import io.zeebe.containers.ZeebeGatewayContainer;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.testcontainers.containers.Network;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+/**
+ * Tests a small deployment of one standalone broker and gateway, running "rootless". While this
+ * cannot guarantee OpenShift compatibility, it's the most common compatibility issue.
+ *
+ * <p>See <a
+ * href="https://docs.openshift.com/container-platform/latest/openshift_images/create-images.html">here</a>
+ * for more.
+ *
+ * <p>From their docs: By default, OpenShift Container Platform runs containers using an arbitrarily
+ * assigned user ID. This provides additional security against processes escaping the container due
+ * to a container engine vulnerability and thereby achieving escalated permissions on the host node.
+ * For an image to support running as an arbitrary user, directories and files that are written to
+ * by processes in the image must be owned by the root group and be read/writable by that group.
+ * Files to be executed must also have "group" execute permissions.
+ *
+ * <p>You can read more about UIDs/GIDs <a
+ * href="https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids">here</a>.
+ */
+@Testcontainers
+final class RootlessImageTest {
+  private final Network network = Network.newNetwork();
+
+  @Container
+  private final ZeebeBrokerContainer broker =
+      new ZeebeBrokerContainer(ZeebeTestContainerDefaults.defaultTestImage())
+          .withNetwork(network)
+          .withCreateContainerCmdModifier(cmd -> cmd.withUser("1000620000:0"));
```

Comment:
The group id 0 refers to root group, isn't it? Can the root group have the privileges similar to root user?
```
I have no name!@3e8708a3a616:/usr/local/zeebe$ id
uid=1000620000 gid=0(root) groups=0(root)
```

Reply:
Possibly, yes. However, this is straight from the OpenShift recommendations, see [here](https://docs.openshift.com/container-platform/4.10/openshift_images/create-images.html#use-uid_create-images)

> For an image to support running as an arbitrary user, directories and files that are written to by processes in the image must be owned by the root group and be read/writable by that group. Files to be executed must also have group execute permissions.
>
> Adding the following to your Dockerfile sets the directory and file permissions to allow users in the root group to access them in the built image:
>
> ```
> RUN chgrp -R 0 /some/directory && \
>     chmod -R g=u /some/directory
> ```
>
> Because the container user is always a member of the root group, the container user can read and write these files.

Further down, they mention it could be a security concern:

> Care must be taken when altering the directories and file permissions of sensitive areas of a container, which is no different than to a normal system. If applied to sensitive areas, such as `/etc/passwd`, this can allow the modification of such files by unintended users potentially exposing the container or host.

Reply:
:+1: